packages/legend-studio/src/stores/LegendStudioPlugin.ts

```diff
@@ -52,6 +53,12 @@ export type ClassPreviewRenderer = (
   _class: Class,
 ) => React.ReactNode | undefined;
 
+export type ExtraModelLoaderRendererConfiguration = {
+  key: string;
+  defaultModelText: string;
```

Comment:
`ExtraModelLoaderRendererConfiguration` -> `ExtraModelLoaderExtensionsConfiguration`
We probably don't need `defaultModelText` as that is specific to external and native types. 
Also it probably make sense to add a `label?: string` property to this as we may not always want to use the key in the render option.